lib/basedriver/capabilities.js

```diff
@@ -51,6 +52,47 @@ function validateCaps (caps, constraints = {}, skipPresenceConstraint = false) {
   return caps;
 }
 
+// Standard, non-prefixed capabilities (see https://www.w3.org/TR/webdriver/#dfn-table-of-standard-capabilities)
+const STANDARD_CAPS = ['browserName', 'browserVersion', 'platformName', 'acceptInsecureCerts', 'pageLoadStrategy', 'proxy', 'setWindowRect', 'timeouts', 'unhandledPromptBehavior'];
+
+function isStandardCap (cap) {
+  return !!_.find(STANDARD_CAPS, (standardCap) => standardCap.toLowerCase() === cap.toLowerCase());
```

Comment:
`${cap}` would be safer

Reply:
Sorry, I don't think I understand. What does `${cap}` replace?

Reply:
I mean that it is not guaranteed that `cap` is of type string, which may cause an unexpected exception.